src/spikeinterface/extractors/neoextractors/neuralynx.py

```diff
@@ -22,22 +22,24 @@ class NeuralynxRecordingExtractor(NeoBaseRecordingExtractor):
         If there are several streams, specify the stream name you want to load.
     all_annotations: bool, default: False
         Load exhaustively all annotations from neo.
+    exlude_filename: list[str], default: None
+        List of filename to exclude from the loading.
     """
 
     mode = "folder"
     NeoRawIOClass = "NeuralynxRawIO"
     name = "neuralynx"
 
-    def __init__(self, folder_path, stream_id=None, stream_name=None, all_annotations=False):
-        neo_kwargs = self.map_to_neo_kwargs(folder_path)
+    def __init__(self, folder_path, stream_id=None, stream_name=None, all_annotations=False, exclude_filename=None):
+        neo_kwargs = self.map_to_neo_kwargs(folder_path, exclude_filename)
         NeoBaseRecordingExtractor.__init__(
             self, stream_id=stream_id, stream_name=stream_name, all_annotations=all_annotations, **neo_kwargs
         )
         self._kwargs.update(dict(folder_path=str(Path(folder_path).absolute())))
```

Comment:
```suggestion
        self._kwargs.update(dict(folder_path=str(Path(folder_path).absolute()), exclude_filename=exclude_filename))
```
This is not a "default" neo argument (like `stream_name` etc.), so it should be appended to the `self._kwargs`